### PR TITLE
Update software requirements for Safari & Firefox

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation update for:
- https://github.com/mattermost/mattermost-server/pull/19271 
- https://github.com/mattermost/mattermost-server/pull/19564

### CODE REVIEW

**Comment by cwarnermm** [comment]
@amyblais - The product documentation currently lists Windows support as 8.1 & 10 only, rather than 8.1+. Please let me know if you'd prefer to see Windows support updated to 8.1+.

**Comment by amyblais** [comment]
> @amyblais - The product documentation currently lists Windows support as 8.1 & 10 only, rather than 8.1+. Please let me know if you'd prefer to see Windows support updated to 8.1+.

Sounds good, I'll add Devin as a reviewer just to double-confirm that we should update this.

**Review by devinbinnie — APPROVED** [review]

File: source/install/software-hardware-requirements.rst [hunk 1]
```diff
@@ -55,11 +55,11 @@ PC Web
     :header: "Browser", "Self-Hosted Technical Requirement", "Cloud Technical Requirement"
```
Comment (amyblais) [inline source/install/software-hardware-requirements.rst:55]: Windows support can be updated to 8.1+.

Comment (cwarnermm) [reply source/install/software-hardware-requirements.rst:55]: Updated! Thanks!

**Review by amyblais — APPROVED** [review]

**Pull request merged**